rskj-core/src/main/java/co/rsk/rpc/netty/Web3ResultHttpResponseHandler.java

```diff
@@ -29,11 +29,15 @@
 
     @Override
     protected void channelRead0(ChannelHandlerContext ctx, Web3Result msg) {
-        ctx.write(new DefaultFullHttpResponse(
-            HttpVersion.HTTP_1_1,
-            HttpResponseStatus.valueOf(DefaultHttpStatusCodeProvider.INSTANCE.getHttpStatusCode(msg.getCode())),
-            msg.getContent()
-        )).addListener(ChannelFutureListener.CLOSE);
+        DefaultFullHttpResponse response = new DefaultFullHttpResponse(
+                HttpVersion.HTTP_1_1,
+                HttpResponseStatus.valueOf(DefaultHttpStatusCodeProvider.INSTANCE.getHttpStatusCode(msg.getCode())),
+                msg.getContent()
+        );
+
+        response.headers().add("Content-Type", "application/json");
```

Comment:
You can use: 
import static io.netty.handler.codec.http.HttpHeaders.Names.CONTENT_TYPE;
import static io.netty.handler.codec.http.HttpHeaders.Values.APPLICATION_JSON;

Reply:
We could also remove unused import:
import io.netty.channel.ChannelFutureListener;

Reply:
Added constants. The ChannelFutureListener import is still in use